LOADERS/CLOUD_DATABASE/LOADER/LOADER.py

```diff
@@ -59,3 +59,17 @@ def get_api_key():
     else:
         api_key = yaml_dict.get("FRONTIER_API_KEY", "")
     return api_key
+
+
+def set_api_key(api_key: str) -> bool:
+    home = str(Path.home())
+    path = os.path.join(home, ".flojoy/credentials")
+
+    if not os.path.exists(path):
+        return False
+
+    stream = open(path, "r", encoding="utf-8")
```

Comment:
Please use the `with` statement to open the file, which will be automatically recycled.